jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash -ex
-REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
-
+if (echo ${REPOS_UNDER_TEST} | grep -q  "image-service"); then
+    REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" "image-service")
+else
+    REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
+fi
```

Comment:
Theoretically it would make the code clear and beautiful,  yet the reality is a little bit skinny, so I'd leave it as it is
tuotuo@sunny-dev:~$ cat test.sh
#!/bin/bash

a=("aaa" "bbb")
a+=("ccc")
echo $a
tuotuo@sunny-dev:~$ ./test
-bash: ./test: Is a directory
tuotuo@sunny-dev:~$ ./test.sh
aaa